### PR TITLE
Gooddollar Hot Fix and logging implementation

### PR DESCRIPTION
**Pull request by harjaapdhillon16**
Fixed the Issues people were facing in Gooddollar FV SBT .

### CODE REVIEW

**Comment by Megha-Dev-19** [comment]
@harjaapdhillon16 Is the changes about the fix working.? I see in the group chats that we still have issue with it
Is that fixed now or are we still waiting for more details about the error?

File: src/components/pages/landing/Gooddollar.js [hunk 1]
```diff
@@ -29,96 +30,104 @@ export const Gooddollar = ({ setShowGooddollarVerification }) => {
...
-    });
+        log_event({ event_log: 'Applied for FV SBT' });
+        await wallet.callMethod({
+          contractId: gooddollar_contract,
+          method: 'sbt_mint',
+          args: {
+            claim_b64: result.m,
+            claim_sig: result.sig,
+          },
+          deposit: '8000000000000000000000',
```
Comment (robert-zaremba) [inline src/components/pages/landing/Gooddollar.js]: this should be a config. Please create a config file with constants we are using.

File: src/components/pages/landing/Gooddollar.js [hunk 2]
```diff
@@ -29,96 +30,104 @@ export const Gooddollar = ({ setShowGooddollarVerification }) => {
...
+  } = useFormik({
+    initialValues: {
+      name: '',
+      gDollarAccount: '',
+      status: '',
+    },
+    validationSchema: Yup.object().shape({
+      email: Yup.string().email('Invalid email'),
+    }),
+    onSubmit: async (data) => {
```
Comment (robert-zaremba) [inline src/components/pages/landing/Gooddollar.js]: this is a long function and should be broken down.

**Review by robert-zaremba — APPROVED** [review]
pre-approved. Would be good to have the code better organized. This PR has indentation changes, making it harder to read what actually has changed.

File: src/components/pages/landing/Gooddollar.js (hunk 1 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/Gooddollar.js]: fixed

File: src/components/pages/landing/Gooddollar.js (hunk 2 above)
Comment (harjaapdhillon16) [reply src/components/pages/landing/Gooddollar.js]: Will do in a separate pr

**Comment by harjaapdhillon16** [comment]
https://github.com/near-ndc/i-am-human-dapp/actions/runs/4917636845/jobs/8783232655?pr=30 @Megha-Dev-19  Can you please check why these tests are failing there were no changes to these files !

**Comment by Megha-Dev-19** [comment]
> https://github.com/near-ndc/i-am-human-dapp/actions/runs/4917636845/jobs/8783232655?pr=30 @Megha-Dev-19 Can you please check why these tests are failing there were no changes to these files !

Fixed

**Pull request merged**